README.rst

```diff
@@ -1023,16 +1023,8 @@ jitter, datagram loss etc of the device using `iperf3 utility <https://iperf.fr/
 This check is **disabled by default**. You can enable auto creation of this check by setting the
 `OPENWISP_MONITORING_AUTO_IPERF <#OPENWISP_MONITORING_AUTO_IPERF>`_ to ``True``.
 
-You can also create the iperf check from the **device page** as shown below:
-
-.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/device-inline-check.png
-  :align: center
-
-**Note:** To access the above page, the user must have some special permissions,
-which are included by default in the groups "Administrator" and "Operator" and are shown in the screenshot below.
-
-.. figure:: https://github.com/openwisp/openwisp-monitoring/raw/docs/docs/1.1/inline-permissions.png
-  :align: center
+You can also `add the iperf check and it's alert settings
+<#add-checks-and-alert-settings-from-the-device-page>`_ directly from the device page.
```

Comment:
I think we should also add a note that even when `OPENWISP_MONITORING_AUTO_IPERF` is set to True, users need to update the metric configuration to enable alerts for the iperf check.